{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/account/logout.html

```diff
@@ -5,18 +5,28 @@
 {% block head_title %}{% trans "Sign Out" %}{% endblock %}
 
 {% block inner %}
+
+{% if ACCOUNT_HIDE_INTERMEDIARY_LOGOUT %}
+  {# Hide form - prevents user confusion by seeing an auto-submitting form #}
+  <script>document.getElementsByClassName('container')[0].style.display = 'none';</script>
+{% endif %}
+
```

Comment:
The document might have multiple `container` so if there is another container (e.g. in the header) this script will fail. I would suggest wrapping the form in a div 

```
<div style='display:none'>
<form ...> </form>
</div>
```
instead.